crates/http-sink/Connector.toml

```diff
@@ -3,7 +3,7 @@ name = "http-sink"
 group = "infinyon"
 version = "0.2.11"
 apiVersion = "0.1.0"
-fluvio = "0.12.0"
+fluvio = "0.13.0"
```

Comment:
will this work with previous version of fluvio? Is there reason to bump this up?

Reply:
I think that this is only metadata of which fluvio version this connector is using. 

Should work with previous fluvio version too.